src/SCE/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+if(ENABLE_SCE)
```

Comment:
I suggest to do this in the root CMakeLists.txt, you are using that pattern for CCE.

Reply:
@mpreisler OK